src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -645,28 +883,31 @@ bool OnOffServer::areStartUpOnOffServerAttributesNonVolatile(EndpointId endpoint
 /**
  * @brief event control object for an endpoint
  *
- * @param[in] endpoint
+ * @param[in] endpoint target endpoint
+ * @param[in] eventControlArray Array where to find the event control
+ * @param[in] eventControlArraySize Size of the event control array
  * @return EmberEventControl* configured event control
  */
-EmberEventControl * OnOffServer::getEventControl(EndpointId endpoint)
+EmberEventControl * OnOffServer::getEventControl(chip::EndpointId endpoint, EmberEventControl * eventControlArray,
```

Comment:
Might have been nicer with `const Span<EmberEventControl> &` as the second arg and no separate size arg.  But maybe not, if the Span constructor that takes a C array is "explicit"...  Either way.